linux_os/guide/system/accounts/accounts-physical/require_emergency_target_auth/ansible/shared.yml

```diff
@@ -4,7 +4,7 @@
 # complexity = low
 # disruption = low
 
-{{% if 'sle' in product %}}
+{{% if 'sle' in product or 'rhel' in product %}}
```

Comment:
I think you should add a test scenario that covers the situation when the `.d` directory is used.